api/nodes/serializers.py

```diff
@@ -375,3 +377,57 @@ class Meta:
     @staticmethod
     def get_id(obj):
         return '{}:{}'.format(obj.node._id, obj.provider)
+
+class NodeAlternativeCitationSerializer(JSONAPISerializer):
+
+    id = IDField(source="_id", read_only=True)
+    name = ser.CharField(read_only=True)
+    text = ser.CharField(read_only=True)
+
+    class Meta:
+        type_ = 'citations'
+
+    def create(self, validated_data):
+        data = self.context['request'].data
```

Comment:
I'm not sure, but if I put a break point and investigate it, it is a blank dictionary.